### DISCUSSION TITLE
Layout Settings

### DISCUSSION THREAD

**Discussion by Nadir920** [Hoogkamer/Termboard#20] [Q&A] [answered]
Hello, your tool is great I like it a lot but I encounter a problem, once I add the pages (from chatgpt) in the mind map, and I click on Layout Settings to organize the plan, the pages do not follow not a logical direction because it overlaps with one another and becomes incomprehensible, I have to create a very large mental map but once I click on organized by selecting "COLA" the pages fit into the plan and its part in all the way, normally when I click on organized the pages should go in the direction of the side where the plan is empty from level to level, and this prevents me from moving forward in the construction of the mindmap. Is there a solution, I can show examples if necessary.

**Comment by Hoogkamer** [comment, marked as answer]
Hi, thank you. It is a challenge with bigger graphs to get the algorithm position the elements in a readable way. You could also try another layout type maybe ( "widest" for example). Or you can try the following steps:

1. When you have added terms/relations using the bulk add, they are automatically selected
2. Do not right-click on the canvas otherwise the elements get deselected
3. Run the layout on the selected elements only. In the layout settings, choose Which elements = Selected
4. Drag 1 selected element to the spot you want, then all selected elements move with it
5. OR go to the spot where you want the elements (by keep pressing the right mouse button and then drag to the spot), then right click to bring up the context menu. Select "Edit Selected ..." and then "Move  here"
